azurerm/resource_arm_log_analytics_linked_service.go

```diff
@@ -0,0 +1,178 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/operationalinsights/mgmt/2015-11-01-preview/operationalinsights"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmLogAnalyticsLinkedService() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmLogAnalyticsLinkedServiceCreateUpdate,
+		Read:   resourceArmLogAnalyticsLinkedServiceRead,
+		Update: resourceArmLogAnalyticsLinkedServiceCreateUpdate,
+		Delete: resourceArmLogAnalyticsLinkedServiceDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"resource_group_name": resourceGroupNameDiffSuppressSchema(),
+
+			"workspace_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
```

Comment:
Could we get some validation here? At the very least `validation.NoZeroValues`

Reply:
Added the validation for the workspace name by using an already existing function.